multigeiger/multigeiger.ino

```diff
@@ -304,6 +304,7 @@ void clearDisplayLine(int line);
 void handleRoot(void);
 void configSaved(void);
 char * nullFill(int n, int digits);
+char* buildSSID();
```

Comment:
can we try to keep style consistent (see line directly above)?

Reply:
OK. Seems 'nullfill' is the only place, where there is the additional spaces.